.env.example

```
@@ -0,0 +1,5 @@
+DATABASE_HOST=127.0.0.1
```

```suggestion
DATABASE_HOST=db
```
chỗ này e thử để là db xem. để như thế thì app vs db ms connect vs nhau qua docker network